lib/voxpupuli/test/rake/validate.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+require 'rspec/core/rake_task'
+require 'metadata-json-lint/rake_task'
+require 'puppet-strings/tasks'
+
+desc 'Check syntax of Ruby files and call :syntax and :metadata_lint'
+task :validate do
+  Dir['lib/**/*.rb'].each do |lib_file|
+    sh ['ruby', '-c', lib_file]
+  end
+
+  Rake::Task[:syntax].invoke
+  Rake::Task[:metadata_lint].invoke
+
+  if File.exist?('REFERENCE.md')
+    if Rake::Task.task_defined?('strings:validate:reference')
+      Rake::Task['strings:validate:reference'].invoke
+    else
+      warn 'Skipping reference documentation validation; the puppet-strings gem was not found'
+    end
```

Comment:
We explicitly pull in puppet-strings as a runtime dependency so we can simplify this
```suggestion
    Rake::Task['strings:validate:reference'].invoke
```